zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -290,9 +295,12 @@ private Stream<String> getUserAuthorizedResourceIdentifiers(
         authorizationState.getResourceIdentifiers(
             AuthorizationOwnerType.USER, user.getUsername(), resourceType, permissionType);
     // Get resource identifiers for the user's roles
+    final var roles =
+        membershipState.getRelations(
```

Comment:
💭 I wonder if `relations` is a meaningful term here, ultimately I'm getting the roles where a user is a member of, so something like `getMemberships()` would also be an option. NABD though

Reply:
I think I prefer `getMemberships` :+1:

Reply:
Changed in c5b9fd4541c0e5e983c360503cd92294bf6abeff